### ISSUE TITLE
Visual indication a cell has been evaluated by Thebe

### ISSUE THREAD

**Issue by jackgerrits** [executablebooks/thebe#540]

**Comment by choldgraf** [comment]
Transferred this one over because I think this is where the fix will be. It might be helpful if the "loading wheel" that thebe uses ended with a green checkmark so that the reader knows that it has completed successfully. What do you think @stevejpurves ?

**Comment by stevejpurves** [comment]
yes, leaving the indicator as a green tick can be added. That would mark the completion of execution, rather than successful execution though?  but we could probably add a green tick / red error depending on the contents of the output easily enough.

**Comment by rcpeene** [comment]
Did this get completed? Using thebe with jupyterbook and would still love this functionality.
